Automattic/vip
# Add codegen for GraphQL queries

## Description

Adds the GraphQL Code generator.

Best to review it by ignoring all commits that says 'Regenerate graphql types'.

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Get a hold of `schema.gql` and paste it in project root - this is the schema of the endpoint that
   we communicate with.
2. Run `npm run typescript:codegen:install-dependencies` - this will install the codegen
   dependencies without updating `package.json`
3. Run `npm run typescript:codegen:generate` - this will regenerate the types
